autopush/router/apns2.py

```diff
@@ -21,6 +22,16 @@
 APNS_PRIORITY_LOW = '5'
 
 
+class ComplexEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, Decimal):
+            return int(float(obj.to_integral()))
```

Comment:
And, this is why reading a code review comment while following a keynote is probably not a good idea. Misread your comment. *sigh*